resolwe/flow/execution_engines/bash/__init__.py

```diff
@@ -0,0 +1,38 @@
+"""An execution engine that outputs bash programs."""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from django.conf import settings
+
+from resolwe.flow.execution_engines import BaseExecutionEngine, ExecutionError
```

Comment:
`bash` is a submodule of `execution_engines`, however we do not import it in `execution_engines` (yet). Could this introduce a circular import in the future?